tests/test_common_cells.hpp

```diff
@@ -54,10 +61,15 @@ inline cell make_cell_ball_and_stick(bool with_stim = true) {
     auto soma = c.add_soma(12.6157/2.0);
     soma->add_mechanism(hh_parameters());
 
-    auto dendrite = c.add_cable(0, segmentKind::dendrite, 1.0/2, 1.0/2, 200.0);
-    dendrite->add_mechanism(pas_parameters());
-    dendrite->mechanism("membrane").set("r_L", 100);
-    dendrite->set_compartments(4);
+    c.add_cable(0, segmentKind::dendrite, 1.0/2, 1.0/2, 200.0);
+
+    for (auto& seg: c.segments()) {
```

Comment:
I earlier missed the setting of `r_L` for the soma segment, and thought this made it clearer that all (admittedly two) segments should have the same `r_L` parameter set.